src/managing-workflow/configuring-load-balancers.md

```diff
@@ -1,15 +1,43 @@
 
-If you want to configure SSL termination on your load balancer, see [Platform SSL](platform-ssl.md).
+If, for instance, Deis Workflow were installed on kube-aws, this timeout should be increased to a recommended value of 1200 seconds.  This will ensure the load balancer does not hang up on the client during long-running operations like an application deployment.  Directions for this can be found [here](http://docs.aws.amazon.com/ElasticLoadBalancing/latest/DeveloperGuide/config-idle-timeout.html).
 
-A health check should be configured on the load balancer to send an HTTP request to /healthz at port 9090 on all nodes in the Deis cluster. The health check endpoint returns an HTTP 200. This enables the load balancer to serve traffic to whichever hosts happen to be running the router component at any moment.
+## Manually configuring a load balancer
 
-See [the router component's own documentation](https://github.com/deis/router#front-facing-load-balancer) for further details.
+If using a Kubernetes distribution or underlying infrastructure that does not support the automated provisioning of a front-facing load balancer, operators will wish to manually configure a load balancer (or use other tricks) to route inbound traffic from beyond the cluster into the cluster to the Deis router(s).  There are many ways to accomplish this.  This documentation will focus on the most common method.  Readers interested in other options may refer to [the router component's own documentation](https://github.com/deis/router#front-facing-load-balancer) for further details.
+
+Begin by determining the "node ports" for the `deis-router` service:
+
+
+```
+$ kubectl describe service deis-router --namespace=deis
+```
+
+This will yield output similar to the following:
+
+```
+...
+Port:			http	80/TCP
+NodePort:		http	32477/TCP
+Endpoints:		10.2.80.11:80
+Port:			https	443/TCP
+NodePort:		https	32389/TCP
+Endpoints:		10.2.80.11:443
+Port:			builder	2222/TCP
+NodePort:		builder	30729/TCP
+Endpoints:		10.2.80.11:2222
+Port:			healthz	9090/TCP
+NodePort:		healthz	31061/TCP
+Endpoints:		10.2.80.11:9090
+...
+```
+
+The node ports shown above are high-numbered ports that are allocated on every Kubernetes worker node for use by the router service.  The kube-proxy component on every Kubernetes node will listen on these ports and proxy traffic through to the corresponding port within an endpoint-- that is, a pod running the Deis router.
+
+If manually creating a load balancer, configure it to have all Kubernetes worker nodes in the back-end pool, and listen on ports 80, 443, and 2222 (port 9090 can be ignored).  Each of these listeners should proxy inbound traffic to the corresponding node ports on the worker nodes.  Ports 80 and 443 may use either HTTP/S or TCP as protocols.  Port 2222 must use TCP.
```

Comment:
I agree this could be worded better, but I think we should avoid "minion."  From what I have heard, Kubernetes is striking that word from their own vernacular in favor of "nodes" or "worker nodes," when differentiation from the master(s) is necessary.  I think we should follow their lead on that.